maxim/tests/test_openai.py

```
@@ -24,7 +24,7 @@ async def asyncSetUp(self):
         # This is a hack to ensure that the Maxim instance is not cached
         if hasattr(Maxim, "_instance"):
             delattr(Maxim, "_instance")
-        self.logger = Maxim().logger()
+        self.logger = Maxim({ "base_url": baseUrl }).logger()
 
     async def test_async_chat_completions(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_async_chat_completions`

Add return type annotation: `None`

(ANN201)